RevenueCatUI/Scripts/RevenueCatUI.cs

```diff
@@ -7,24 +7,24 @@ namespace RevenueCat.UI
     /// <summary>
     /// Main interface for RevenueCat UI components.
     /// Provides methods to present paywalls.
+    /// Add this component to a GameObject to enable paywall functionality.
     /// </summary>
-    public static class RevenueCatUI
+    public class RevenueCatUI : MonoBehaviour
```

Comment:
```suggestion
    public sealed class RevenueCatUI : MonoBehaviour
```

I am not 100% sure about it, but feels like this class shouldn't be subclassed